src/hotspot/share/classfile/classFileParser.cpp

```diff
@@ -5576,8 +5576,8 @@ ClassFileParser::~ClassFileParser() {
   }
 
   if (_methods != nullptr) {
-    // Free methods
-    InstanceKlass::deallocate_methods(_loader_data, _methods);
+    // Free methods - those methods are not fully wired and miss the method holder
```

Comment:
How about saying: for methods whose InstanceKlass as method holder is not yet created?

Reply:
This comment went away with change @dholmes-ora recommended